app/controllers/admin/routes.py

```diff
@@ -430,12 +430,17 @@ def updatecohort(year, method, username):
     if method == "add":
         try:
             BonnerCohort.create(year=year, user=user)
+            flash("Sucessfully added "+user.fullName+" to Bonner Cohorts", "success")
```

Comment:
Consider adding spacing between variables and symbols. It is a good coding practice and it looks cleaner and more organized that way. This comment is applicable to all the flash messages you added

Reply:
I added spacing between the plus signs, better separating the variables to make it more readable.